Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -609,15 +609,20 @@ extension GridTabViewController {
         openNewTab(isPrivate: tabDisplayManager.isPrivate)
     }
 
-    func didTapToolbarDelete(_ sender: UIBarButtonItem) {
+    func didTapToolbarDelete(_ sender: Any) {
         if tabDisplayManager.isDragging {
             return
         }
 
         let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
         controller.addAction(UIAlertAction(title: Strings.AppMenuCloseAllTabsTitleString, style: .destructive, handler: { _ in self.closeTabsForCurrentTray() }), accessibilityIdentifier: "TabTrayController.deleteButton.closeAll")
         controller.addAction(UIAlertAction(title: .TabTrayCloseAllTabsPromptCancel, style: .cancel, handler: nil), accessibilityIdentifier: "TabTrayController.deleteButton.cancel")
-        controller.popoverPresentationController?.barButtonItem = sender
```

Comment:
FYI: at this point sender was a `UIButton` (UIBarButtonItem with custom UIButton view) and not `UIBarButtonItem` which caused the crash 💥